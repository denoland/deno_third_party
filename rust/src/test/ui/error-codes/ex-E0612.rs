// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
// file at the top-level directory of this distribution and at
// http://rust-lang.org/COPYRIGHT.
//
// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
// option. This file may not be copied, modified, or distributed
// except according to those terms.

struct Foo(u32);

fn main() {
   let y = Foo(0);
   y.1; //~ ERROR no field `1` on type `Foo`
}
