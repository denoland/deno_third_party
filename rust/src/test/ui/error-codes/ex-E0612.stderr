error[E0609]: no field `1` on type `Foo`
  --> $DIR/ex-E0612.rs:15:6
   |
LL |    y.1; //~ ERROR no field `1` on type `Foo`
   |      ^ did you mean `0`?

error: aborting due to previous error

For more information about this error, try `rustc --explain E0609`.
